cli/tests/pcluster/config/test_section_fsx.py

```diff
@@ -35,10 +38,13 @@
                 "weekly_maintenance_start_time": None,
                 "deployment_type": None,
                 "per_unit_storage_throughput": None,
+                "daily_automatic_backup_start_time": None,
+                "automatic_backup_retention_days": None,
+                "copy_tags_to_backups": False,
```

Comment:
I know you're extending an existing test case, but if it's alright with you let's take the opportunity to make this test code a bit more succinct.

Instead of writing out the entire expected dict, you can use `utils.merge_dicts(DefaultDict["fsx"].value, {"shared_dir": "test"})`.